internal/buildscripts/packaging/installer/install.sh

```diff
@@ -392,6 +419,14 @@ install() {
       if [ -n "$td_agent_version" ]; then
         install_td_agent_yum_repo "$td_agent_version"
         install_yum_package "td-agent" "$td_agent_version"
+        if command -v yum >/dev/null 2>&1; then
+          yum group install -y 'Development Tools'
+        else
+          dnf group install -y 'Development Tools'
+        fi
+        for pkg in libcap-ng libcap-ng-devel pkgconfig; do
```

Comment:
is `libcap-ng` required?

Reply:
See https://github.com/signalfx/splunk-otel-collector/pull/139#discussion_r585830676